app.js

```diff
@@ -29,12 +31,13 @@ async function load() {
 // This function is used to change the page. 
 // It gets new char content and renders it in the grid.
 async function page(event) {
-    console.log(currentPage);
-    targetID = event.target.id;
-    currentPage = targetID === 'next-page' ? currentPage + 1 : currentPage - 1;
-    if (currentPage < 1) { currentPage = 1; }
-    await getData(`?page=${currentPage}`, 'grid');
-    await showData('grid');
+    let targetID = event.target.id; 
+    if(paginationCheck(targetID)) {
+        currentPage = targetID === 'next-page' ? currentPage + 1 : currentPage - 1;
+        await getData(`?page=${currentPage}`, 'grid');
+        await showData('grid');
+    } 
+
```

Comment:
```suggestion
    let targetID = event.target.id; 
    if (!paginationCheck(targetID)) {
        return;
    }
    
    currentPage = targetID === 'next-page' ? currentPage + 1 : currentPage - 1;
    await getData(`?page=${currentPage}`, 'grid');
    await showData('grid');
```

It's better to have an early return than to wrap the contents of a function inside of a condition. 

One way you can keep an eye out for this is that the end of the code block after the `if` is also the end of your function. 

Example of what not to do:
```js
function something(event) {
    const myCondition = event.condition
    if (myCondition) {
        // All of the contents of the something function are in here
        code
        code
        code
    } // end of code block for if statement
} // end of code block
```

Example of what to do:
```js
function something(event) {
    const myCondition = event.condition
    if (!myCondition) {
        return; // early return 
    }
    
    // rest of the function
    code
    code
    code
}
```

Reply:
Noted -- that makes sense. This a good example of how to make my code flatter which is definitely something I appreciate tips for.